### ISSUE TITLE
Failure to parse RSS feed with CDATA in copyright field

### ISSUE THREAD

**Issue by sgrove** [Cumulus/Syndic#76]
Trying to parse `https://16andcounting.libsyn.com/rss` fails with `"Error parsing feed: The content of <copyright> MUST be a non-empty string"`

The actual copyright tag looks like: `<copyright><![CDATA[]]></copyright>`

**Comment by dinosaure** [comment]
Indeed, W3C does not tell me an error about that. You are right, I will fix it.
